src/components/organisms/rma-select-receive-product-table/index.tsx

```diff
@@ -0,0 +1,182 @@
+          const checked = item.id in toReturn
+          return (
+            <Table.Row className={clsx("border-b-grey-0 hover:bg-grey-0")}>
+              <Table.Cell>
+                <div className="items-center ml-1 h-full flex">
+                  <div
+                    onClick={() => handleReturnToggle(item)}
+                    className={`mr-4 w-5 h-5 flex justify-center text-grey-0 border-grey-30 border cursor-pointer rounded-base ${
+                      checked && "bg-violet-60"
+                    }`}
+                  >
+                    <span className="self-center">
+                      {checked && <CheckIcon size={16} />}
+                    </span>
+                  </div>
+
+                  <input
+                    className="hidden"
+                    checked={checked}
+                    tabIndex={-1}
+                    onChange={() => handleReturnToggle(item)}
+                    type="checkbox"
+                  />
+                </div>
+              </Table.Cell>
+              <Table.Cell>
+                <div className="min-w-[240px] flex py-2">
+                  <div className="w-[30px] h-[40px] ">
+                    <img
+                      className="h-full w-full object-cover rounded"
+                      src={item.thumbnail}
+                    />
+                  </div>
+                  <div className="inter-small-regular text-grey-50 flex flex-col ml-4">
+                    <span>
+                      <span className="text-grey-90">{item.title}</span>
+                    </span>
+                    <span>{item?.variant?.title || ""}</span>
+                  </div>
+                </div>
+              </Table.Cell>
+              {/* <Table.Cell className="text-right w-32 pr-8">
+                <span className="text-grey-40">{item.quantity}</span>
+              </Table.Cell> */}
+              <Table.Cell className="text-right w-32 pr-8">
+                {item.id in toReturn ? (
+                  <div className="flex w-full text-right justify-end text-grey-50 ">
+                    <span
+                      onClick={() => handleQuantity(-1, item)}
+                      className="w-5 h-5 flex items-center justify-center rounded cursor-pointer hover:bg-grey-20 mr-2"
+                    >
+                      <MinusIcon size={16} />
+                    </span>
+                    <span>{toReturn[item.id].quantity || ""}</span>
+                    <span
+                      onClick={() => handleQuantity(1, item)}
+                      className={clsx(
+                        "w-5 h-5 flex items-center justify-center rounded cursor-pointer hover:bg-grey-20 ml-2"
+                      )}
+                    >
+                      <PlusIcon size={16} />
+                    </span>
+                  </div>
+                ) : (
+                  <span className="text-grey-40">
+                    {item.quantity - item.returned_quantity}
```

Comment:
It's possible to make this return a negative number. As an example if you have an line item with a quantity of 4, and you first make a return on two of them and then go to create a new return on a third of them, then the item that is being displayed will look like this: 

```json
...
quantity: 1
refundable: 7300
returned_quantity: 2
shipped_quantity: 4
...
```
Because the quantity here is 1 (the requested return quantity) and there has already been 2 items returned, it will then display `-1`